pom.xml

```diff
@@ -61,6 +61,18 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
+                <groupId>javax.annotation</groupId>
+                <artifactId>javax.annotation-api</artifactId>
+                <version>1.3.1</version>
+            </dependency>
+
+            <dependency>
+                <groupId>com.sun.xml.bind</groupId>
```

Comment:
This managed dependency not necessary, there's already the glassfish JAXB impl on the classpath. Should be removed.